userguide/storage.rst

```diff
@@ -217,13 +217,12 @@ Click the *+* next to the volume name to access
 Encryption
 ^^^^^^^^^^
 
-.. note:: The encryption facility used by %brand% is designed to
-   protect against physical theft of the disks. It is not designed to
-   protect against unauthorized software access. Ensure that only
-   authorized users have access to the administrative GUI and that
-   proper permissions are set on shares if sensitive data is stored on
-   the system.
-
+.. note:: The encryption facility used by %brand% is designed to protect
+   against physical theft and removal of disks, and to prevent data recovery 
```

Comment:
This sentence is kind of contradictory and confusing.  Encryption is only used to prevent data recovery, so leaving the original "theft and removal" in place is not useful.

Reply:
reworded - done